2022q2/pantheon.adoc

```diff
@@ -0,0 +1,24 @@
+=== Pantheon desktop on FreeBSD
+
+Links: +
+link:https://elementary.io/[elementary OS] URL: link:https://elementary.io/[https://elementary.io] +
+link:https://codeberg.org/olivierd/freebsd-ports-elementary[Development repository] URL: link:https://codeberg.org/olivierd/freebsd-ports-elementary[https://codeberg.org/olivierd/freebsd-ports-elementary] +
+
+Contact: Olivier Duchateau <duchateau.olivier@gmail.com>
+
+The Pantheon desktop environment is designed for elementary OS. It builds on GNOME technologies (such Mutter, GNOME Shell, GTK 3 and 4) and it is written with Vala language.
+
+The goal is to have new desktop for users. Some features are not well supported, but we can have full session.
```

Comment:
"have a new desktop"
"can have a full session" (also, not sure what you mean by "full session" - can you clarify?)

Reply:
Even if desktop environment is not supported by upstream, it is daily usable.